addon/templates/components/uni-mobile-number.hbs

```diff
@@ -1,12 +1,15 @@
 {{uni-select
+  class="uni-mobile-number__select-wrapper"
   placeholder=selectPlaceholder
   selected=language
   options=options
+  useAlias=true
```

Comment:
Not in this case, uni-select allows that option to be optional, because it's meant to be available.
But uni-mobile-number is not supposed to be any other way than this, (and I'm not the one who decided this) so it doesn't make sense to prepare the component for something it's going to do. 
I would have to prepare css and also the countries object. Right now it doesn't justify the effort.